build/Program.fs

```diff
@@ -59,7 +61,7 @@ let buildCraftSchema() =
         then failwith "Could not build generated CraftCMS"
 
 let buildGithub() =
-    if Shell.Exec(Tools.dotnet, "run -- --config ../samples/github/snowflaqe.json --generate", path [ solutionRoot; "src" ]) <> 0
+    if Shell.Exec(Tools.dotnet, "run -p Snowflaqe.fsproj -- --config ../samples/github/snowflaqe.json --generate", path [ solutionRoot; "src" ]) <> 0
```

Comment:
Why the addition of `-p Snowflaqe.fsproj` here? The command is executed inside the `./src` directory so it should pick up the project just fine

Reply:
Snowflaqe.fsproj may remain after past tests and the run command will not know which project or solution to use.

Reply:
Maybe clean fixed that issue?